doc/source/visualization.rst

```diff
@@ -221,16 +230,20 @@ To get horizontal bar plots, pass ``kind='barh'``:
 .. ipython:: python
 
    @savefig barh_plot_stacked_ex.png
-   df2.plot(kind='barh', stacked=True);
+   df2.plot.barh(stacked=True);
 
 .. _visualization.hist:
 
 Histograms
 ~~~~~~~~~~
 
+.. versionadded:: 0.17.0
+
+Histogram can be drawn by using the :meth:`DataFrame.plot.hist` and :meth:`Series.plot.hist` methods.
+
 .. versionadded:: 0.15.0
 
-Histogram can be drawn specifying ``kind='hist'``.
+Histogram can be drawn by using the ``plot(kind='hist')`` method.
```

Comment:
i would remove the 0.15.0 comment as you have the 0.17.0 tag here with the new way